Gemfile.lock

```diff
@@ -81,7 +81,7 @@ GEM
     daemons (1.2.2)
     descendants_tracker (0.0.4)
       thread_safe (~> 0.3, >= 0.3.1)
-    dotenv (2.0.1)
+    dotenv (0.7.0)
```

Comment:
As long as the API hasn't changed and it doesn't cause it to explode, no, it shouldn't matter. The 2.0 API was the same as 1.0, so it should be fine... probably worth a quick go locally though! :)